packages/blockchain-api/package.json

```diff
@@ -23,6 +23,7 @@
     "bignumber.js": "^7.2.0",
     "dotenv": "^6.1.0",
     "express": "^4.16.4",
+    "firebase-admin": "^8.8.0",
```

Comment:
Please use the same version we're using in other monorepo packages (8.6.1)

Reply:
:+1: